<?xml version="1.0" ?>
<launch>
  <arg name="command_topic" default="base/twist_mux/command_teleop_keyboard"/>

  <node pkg="cob_teleop" type="cob_teleop_keyboard" name="teleop_keyboard" output="screen">
    <remap from="cmd_vel" to="$(arg command_topic)" />
    <param name="walk_vel" value="0.1" />
    <param name="run_vel" value="0.2" />
    <param name="yaw_rate" value="0.1" />
    <param name="yaw_run_rate" value="0.2" />
  </node>
</launch>
